packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/send/dq-requests.ts

```diff
@@ -47,27 +45,15 @@ export async function sendSignedDQRequests({
       };
       //eslint-disable-next-line @typescript-eslint/no-explicit-any
     } catch (error: any) {
-      const msg = "HTTP/SSL Failure Sending Signed DQ SAML Request";
+      const msg = "Failure Sending SAML Request";
+      const errorString: string = errorToString(error);
+      const errorDetails = error?.response?.data
+        ? `, error details: ${JSON.stringify(error?.response?.data)}`
```

Comment:
nit: no need to question mark as we're already checking if present at the start of the ternary